pkg/flag/db_flags.go

```diff
@@ -30,7 +30,7 @@ var (
 		Aliases: []Alias{
 			{
 				Name:       "skip-update",
-				Deprecated: true, // --security-update was renamed to --skip-db-update
+				Deprecated: true, // --skip-update was renamed to --skip-db-update
```

Comment:
This comment is for the `--skip-update` flag, so I'm not sure why it was referencing the `--security-update` flag.